Sources/Extensions/Maker+Configurations.swift

```diff
@@ -20,26 +20,38 @@ postfix operator >>
 /// - returns: `Maker` instance for chaining relations.
 
 public postfix func << (view: UIView) -> Maker {
-    let maker = Maker(view: view)
+    let maker = Maker(view: .view(view))
     maker.newRect = view.frame
     return maker
 }
 
+/// Start frame configuration for `DEFAULT_STATE` state.
+///
+/// - note: Don't forget to call `>>` operator for ending of configuration.
+///
+/// - parameter layer: The layer you are configuring.
+///
+/// - returns: `Maker` instance for chaining relations.
+
+public postfix func << (layer: CALayer) -> Maker {
+    let maker = Maker(view: .layer(layer))
+    maker.newRect = layer.frame
+    return maker
+}
+
 /// End frame configuration.
 
 public postfix func >> (maker: Maker) {
-    if (maker.view.nx_state as? String) == DEFAULT_STATE {
-        maker.configureFrame()
-    }
+    maker.configureFrame()
 }
 
+public typealias UIViewInstallerBlock = (UIViewMaker) -> Void
```

Comment:
Is it possible to remove `UIViewInstallerBlock`?

Reply:
can not find the reasonable purpose of removing